src/pkg/packager/deploy.go

```diff
@@ -305,7 +305,13 @@ func (p *Packager) getUpdatedValueTemplate(component types.ZarfComponent) (value
 
 	state, err := p.cluster.LoadZarfState()
 
-	// If no distro the zarf secret did not load properly
+	// YOLO mode but no state found so just fake it this run
+	if p.cfg.Pkg.Metadata.YOLO && state.Distro == "" {
```

Comment:
If there is state here( ie. the cluster has been initialized with zarf), this function doesn't catch that and continues on to complete the package deployment. This can often lead to issues since the zarf-agent will mutate any urls to images that YOLO package is trying to use. 

Is that what we want, (acceptable risk given the whole 'yolo' namesake)?

Reply:
One thing we can potentially play around with is pre-adding the `zarf.dev/agent=ignore` label to yolo packages if they specify their namespaces?

Reply:
So thought about this some more, I think it's mixing too many concerns, and we won't support a hybrid YOLO + initialize cluster for now. If they deploy some YOLO things and then later run `zarf init`, we already handle this by the fact that existing namespaces are exempted from the Zarf Agent. I'll add a warning about running a YOLO package on a Zarf cluster as a balance.